jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -117,7 +119,13 @@ public void pointsAreWrittenToInfluxDb() throws Exception {
 		assertThat(points).hasSize(1);
 
 		Point point = points.get(0);
-		assertThat(point.lineProtocol()).startsWith(lineProtocol);
+		String pointLineProtocol = point.lineProtocol();
+		assertThat(pointLineProtocol).startsWith(lineProtocol);
+		// JMX port is written as field by default :
+		// cf Point.lineProtocol() : there is a space after the tags and one after the fields
+		String[] split = pointLineProtocol.split(" ");
+		assertThat(split).hasSize(3);
+		assertThat(split[1]).contains(JMX_PORT_KEY + "=" + DEFAULT_PORT);
```

Comment:
Does it matter that JMX_PORT_KEY is at position `1`? Or should we just test that there is an entry for it?